framework/audio/devtools/audioenginedevtools.h

```diff
@@ -21,50 +21,81 @@
 
 #include <QObject>
 #include <QTimer>
-
+#include <optional>
 #include "modularity/ioc.h"
 #include "iaudioengine.h"
-#include "iaudioplayer.h"
+#include "imidiplayer.h"
 #include "context/iglobalcontext.h"
-#include "internal/sinesource.h"
-#include "internal/midisource.h"
 #include "async/asyncable.h"
+#include "global/iinteractive.h"
+#include "iaudiostream.h"
+#include "rpc/sequencer.h"
```

Comment:
Better to rename the 'rpc' folder. May be confused with the rpc module.

Reply:
@Eism What would be your suggestion?

Reply:
I guess just move to internal

Reply:
moved and name changed